contrib/terraform/techdocs-s3-storage/terraform.tf

```diff
@@ -0,0 +1,73 @@
+#==========================
+# Variables
+#==========================
+
+variable "backstage" {
+  default = "backstage"
+}
+
+#==========================
+# Bucket
+#==========================
+
+resource "aws_s3_bucket" "backstage" {
+  bucket = var.backstage
```

Comment:
Bucket names are globally unique right, not at account level? Wondering if we need a better name here for this variable and maybe use another. Otherwise we're going to have `iam` users called `backstage_bucket_for_my_corp` :)

Reply:
Nice catch! 🙂 Is https://github.com/backstage/backstage/pull/4918/commits/f22a8edaafcba019cf4dcc3b53e0e56a81f3590c enough? :)

Reply:
@benjdlambert can be the judge, but i suggest you get that in the PR @JacobValdemar so we can easily review and merge if found good

Reply:
It is included in the PR :)

Reply:
@JacobValdemar no it's not 😅

Looks good otherwise though, so please add it in :grin: